CHANGELOG_iOS.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+## [1.63.183](https://github.com/brave/brave-browser/releases/tag/v1.63.183)
+
+### Web3  
```

Comment:
nit: tab/white space in front of `Web3`

![image](https://github.com/brave/brave-browser/assets/2602313/3cd65313-0a3b-4025-b0bc-c0c7a26ddb90)

Reply:
Got it, nice catch.